### PR TITLE
Removed a few linter warnings

### PR DESCRIPTION
**Pull request by rodericj**
This is built off of the spacing PR. Once that one is merged in I'll redo this one to send it to development. Or if this gets merged they can go together as one package.

### CODE REVIEW

File: SayTheirNamesTests/DateFormatterServiceTest/DateFormatterServiceTests.swift [hunk 1]
```diff
@@ -14,15 +14,21 @@ class DateFormatterServiceTests: XCTestCase {
     let dateFormatterService = DateFormatterService()
     
     func testDateOfBirthFormat() {
-        let dob = Date.dateFrom(year: 2020, month: 06, day: 03)!
```
Comment (hybridcattt) [inline SayTheirNamesTests/DateFormatterServiceTest/DateFormatterServiceTests.swift:17]: here `!` can simply be removed - `XCTAssertEqual` handles nil and throwing out of the box, by design

**Comment by hybridcattt** [comment]
This got auto-closed because the base branch was deleted. Rebase on top of development and re-push should fix it

File: SayTheirNamesTests/DateFormatterServiceTest/DateFormatterServiceTests.swift (hunk 1 above)
Comment (rodericj) [reply SayTheirNamesTests/DateFormatterServiceTest/DateFormatterServiceTests.swift:17]: @hybridcattt yeah but the thing is the dob isn't put into `XCAssertEqual`. The func that is called does not accept the nil

**Comment by rodericj** [comment]
It got weird in git. The rebase conflicted like crazy so I cherry-picked after rebasing then created a new one. Not my finest moment: https://github.com/Say-Their-Name/say-their-names-ios/pull/53

File: SayTheirNamesTests/DateFormatterServiceTest/DateFormatterServiceTests.swift (hunk 1 above)
Comment (hybridcattt) [reply SayTheirNamesTests/DateFormatterServiceTest/DateFormatterServiceTests.swift:17]: riiiiight, that's not the thing being tested. cool then 👍

**Pull request closed without merging**